README.md

```diff
@@ -1,2 +1,126 @@
+		single_visit_sample_{dataset}_{date}.avro
+```
+
+### Alert packets
+
+Alerts are provided as Snappy-compressed [Avro](https://avro.apache.org/) files.
+The schema used to store the alerts is included in the file.
+
+As the complete sample is about 1 TB, We include two smaller subsets--alerts from a single CCD-visit, and all of the alerts from all CCDs in a single visit.
+
+### Database
+In addition to accessing alert packets via community brokers, during the LSST survey science users with data rights will be able to query the relational Prompt Products Database (PPDB).  
+The contents of the alerts are taken directly from the PPDB `DIASource`, `DIAForcedSource`, `DIAObject`, and `SSObject` records.
+For some use cases direct queries to the PPDB may be more efficient than processing alert packets.
+
+To allow users to explore these different modes of data access, we are also providing the PPDB (`association_{dataset}_{date}.db`) from which the sample alerts were generated as an [SQLite](https://www.sqlite.org) database.
+The SQLite database can be queried using SQL through a variety of interfaces.
+
+Note that in operations, users will query the PPDB using [ADQL](http://www.ivoa.net/documents/ADQL/) via a [TAP service](http://www.ivoa.net/documents/TAP/).
+
+### Software
+Users should install [lsst.alert.packet](https://github.com/lsst/alert_packet) to parse alerts and optionally [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) and [alert-stream](https://github.com/lsst-dm/alert_stream) if they wish to simulate a Kafka alert stream.
+
+## External References and Additional Resources
+
+The [Data Products Definition Document (DPDD)](http://ls.st/dpdd) provides a change-controlled guide to what scientific contents are planned for alerts and other Rubin Observatory data products.
+
+[LDM-612](http://ls.st/LDM-612) describes "Plans and Policies for LSST Alert Distribution," including a high-level overview of the relevant components, a summary of data rights issues, and extensive discussion of the role and selection of community alert brokers.
+
+[DMTN-093](https://dmtn-093.lsst.io/) presents the technical design of the alert distribution system.
+
+[DMTN-102](https://dmtn-102.lsst.io/) summarizes "Key Numbers" for alerts, estimating numbers, sizing, etc.
+
+[lsst.alert.packet](https://github.com/lsst/alert_packet) is a Python package that contains alert schemas as well as utilities for reading and writing Rubin alert packets.
+
+Community broker authors will be interested in simulating the Kafka alert stream itself.  [alert-stream-simulator](https://github.com/lsst-dm/alert-stream-simulator/) provides the relevant code; the overall design is discussed in 
+[DMTN-149](https://dmtn-149.lsst.io/).
+[alert-stream](https://github.com/lsst-dm/alert_stream) currently contains additional code for converting from the [Confluent Wire Format](https://dmtn-093.lsst.io/#management-and-evolution) used in the Kafka stream.
```

Comment:
Underscore vs hypen in the package name.

(It's unfortunate that we have alert-stream-simulator but alert_stream... 🤷‍♂️)